src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(",");
+                String operation = parts[0];
+                int amount = Integer.parseInt(parts[1]);
+                if ("supply".equals(operation)) {
+                    totalSupply += amount;
+                } else if ("buy".equals(operation)) {
+                    totalBuy += amount;
```

Comment:
The variable names 'totalSupply' and 'totalBuy' are informative, but 'operation' and 'amount' could be more descriptive, such as 'transactionType' and 'transactionAmount'.